.github/workflows/test-job.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
         working-directory: ${{env.working-directory}}
 
       - name: Test
-        run: go test ./...
+        run: go test ./pkg/... ./cmd/... -race -parallel 4
```

Comment:
@kevin85421 let me know if this change is acceptable. If not made the e2e test would execute, and fail as there's no kind cluster setup at this point of the pipeline.

Reply:
Can we use `make test` instead?

Reply:
Technically, I don't see a reason why make test cannot be here. I did not add it as it would have been different than what is happening for the other steps in the CI pipeline and I did not seem to be a changed related to the main scope of the PR.

I would prefer that we change the CI (Git hub actions) for the API Server in separate PR and perhaps integrate the execution of the e2e tests with build kite. 

I would not change more in the GH action than I have already done.

Reply:
Sounds good